src/briefcase/commands/new.py

```diff
@@ -81,7 +81,9 @@ def make_class_name(self, formal_name):
         :param formal_name: The formal name
         :returns: The app's class name
         """
-        class_name = re.sub('[^0-9a-zA-Z_]+', '', formal_name)
+        # class_name = re.sub('[^0-9a-zA-Z_]+', '', formal_name)  # creates an empty string for non-ascii
+        class_name = re.sub('\s', '', formal_name)  # changed reg expression to remove spaces
```

Comment:
This won't be sufficient; "123 app" is a legal formal name, but `123app` won't be a legal class name.